docs/types.md

```diff
@@ -84,6 +81,16 @@ The following values are allowed:
  - Negative Email - An email address preceded by a minus sign (e.g., "-alice@test.edulinq.org")
    will remove this address from the email recipients (even if they are not currently there).
    This can be useful when using course roles but you want to exclude someone.
+ - Negative Course Role - A course role preceded by a minus sign (e.g., "-student")
+   will remove all course users with that role. This can be useful when using the "\*"
+   but you want to exclude a role.
+
+### Email
+
+Underlying Type: String
+
+An email address.
```

Comment:
Put in a sentence here about how an email address is usually used to identify a user on the server.